scripts/test/unit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -eu -o pipefail
 
-go test -tags daemon -v $@
+go test -tags daemon -v "$@"
```

Comment:
This surprised me because I thought we wanted the word splitting to happen here, but I guess this is an array? 

The docs confirm that "$@" is actually correct, so :+1: